pinc/iso_lang_list.inc

```diff
@@ -56,6 +56,43 @@ function get_languages_with_dictionaries() {
     return $returnArray;
 }
 
+// Given a langcode, return the Unicode script that language is written in.
+// Will return "Latin" if the langcode is not known.
```

Comment:
Would it be slightly clearer to say 'Will return "Latin" for any langcode that doesn't have an entry in `$langcode_script_map`'?

Reply:
It would be clearer for this implementation but less useful in the future if the implementation changes and the comment isn't updated (which happens surprisingly often). The function is pretty simple to begin with so I don't think this is necessary here.